src/main/java/com/twingly/search/Query.java

```diff
@@ -0,0 +1,157 @@
+package com.twingly.search;
+
+import com.twingly.search.exception.TwinglyException;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * The type Query.
+ */
+public class Query {
+    private static final String BASE_URL = "http://api.twingly.com";
```

Comment:
Year, should, but actually I have no luck to make it working, there was smth with SSL setup, but I cannot say right now what there was exactly, cause my laptop is switched off and is at home :-(